test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -21,23 +18,34 @@
 
 from torch._subclasses import fake_tensor
 from torch.onnx._internal import _beartype, diagnostics, fx as fx_onnx
+from torch.onnx._internal.exporter import ExportOptions, ExportOutput
+from torch.onnx._internal.exporters import (
+    DynamoOptimizeExporter,
+    FXSymbolicTraceExporter,
```

Comment:
Maybe we should discuss this further. It wasn't the original intention to 1) publicly expose different exporter class 2) especially on  v1 which has so limited functionality

If different behaviors are needed already, we have to check whether we can refactor the current poc to remove this. Also, some examples may require features we are not delivering right now, so they might keep using current approach or even be disabled until we figure them out

Reply:
As we discussed, we should be able to consolidate the two dynamo approaches (export vs optimize) and keep symbolic tracing separate for now. Tests will clean up and parameterize in subsequent PRs. Baby steps.